features/buyer/catalogue.feature

```diff
@@ -19,53 +19,39 @@ Scenario: User can click through to g-cloud page
 
 Scenario: User can get the PaaS search results
   Given I am on the /g-cloud page
-  When I click 'Platform as a Service'
+  When I have a random g-cloud lot from the API
+  When I click that lot.name
   Then I am on the 'Search results' page
-  And I see the 'Platform as a Service' breadcrumb
-  And I see a search result
-
-Scenario: User can get the SaaS search results
-  Given I am on the /g-cloud page
-  When I click 'Software as a Service'
-  Then I am on the 'Search results' page
-  And I see the 'Software as a Service' breadcrumb
-  And I see a search result
-
-Scenario: User can get the IaaS search results
-  Given I am on the /g-cloud page
-  When I click 'Infrastructure as a Service'
-  Then I am on the 'Search results' page
-  And I see the 'Infrastructure as a Service' breadcrumb
+  And I see that lot.name breadcrumb
   And I see a search result
 
 Scenario: User is able to search by service id and have result returned.
-  Given I am on the /g-cloud page
+  Given I am on the /g-cloud/search page
   And I have a random g-cloud service from the API
   When I enter that service.id in the 'q' field
-  And I click 'Show services'
+  And I click 'Filter'
   Then I see that service.id in the search summary text
   And I see that service.id as the value of the 'q' field
   And I see that service in the search results
 
 Scenario: User is able to search by service name and have result returned.
-  Given I am on the /g-cloud page
+  Given I am on the /g-cloud/search page
   And I have a random g-cloud service from the API
```

Comment:
For the smoketests I think checking one lot per run is fine.  If there are differences between lots then the longer functional test scenarios should cover all these cases.